packages/sdk-react/src/components/misc/FilterContainer.tsx

```diff
@@ -1,35 +1,45 @@
 import { ReactNode } from 'react';
 
 import { classNames } from '@/utils/css-utils';
-import { Stack } from '@mui/material';
+import { Stack, SxProps } from '@mui/material';
+
+import { Theme } from 'mui-styles';
 
 interface FilterContainerProps {
   className?: string;
   searchField?: ReactNode;
   children: ReactNode;
+  sx?: SxProps<Theme>;
```

Comment:
It seems that changes to FilterContainer aren't necessary because SummaryCardsFilters doesn't use this component.